packages/zoe/src/contracts/mintAndSellNFT.js

```diff
@@ -1,101 +1,117 @@
-/* eslint-disable no-use-before-define */
+// /* eslint-disable no-use-before-define */
```

Comment:
I think something happened to this comment. We might be able to remove it entirely too.

Reply:
Yes, I noticed that it could probably be removed, and commented it out in order to verify, then forgot to finish the job.